src/main/java/net/sf/mzmine/modules/peaklistmethods/peakpicking/deconvolution/DeconvolutionTask.java

```diff
@@ -59,6 +61,11 @@
     // User parameters
     private final ParameterSet parameters;
 
+	private RSessionWrapper rSession;
+	private String errorMsg;
+	private boolean userCanceled;

```

Comment:
Maybe. This is not the same meaning. I think this variable is used in BaselineDeconvPreviewSetup, in case the user presses escape, or else. Anyway, in the context of a preview is available for the given feature, this variable is necessary to tell: "the user asked for cancelling the task" without triggering the Event (=calling @override cancel()). I should probably look deeper into this, because it might be something that was necessary at a time but is not anymore due to Model changes, or else...

But, "userCanceled" and "isCanceled()" are two different things.